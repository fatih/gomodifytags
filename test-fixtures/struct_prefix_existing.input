package foo

type foo struct {
	bar       string
	timestamp time.Time `gaum:"@timestamp"`
}
