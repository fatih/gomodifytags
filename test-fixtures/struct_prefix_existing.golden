package foo

type foo struct {
	bar       string    `gaum:"field_name=bar"`
	timestamp time.Time `gaum:"field_name=@timestamp"`
}
